README.md

```diff
@@ -116,7 +118,12 @@ If you get `fastboot: error: Couldn't parse partition size '0x'` try one of the
 
 - `fastboot flash:raw boot lk2nd.img`
 - `fastboot boot lk2nd.img`, then `fastboot flash lk2nd lk2nd.img`
-
+### ONLY FOR SM-A700H:
+1. You need to flash the image in recovery.
+2. Then restart the phone. If it is stock boot.img in device,then it may loop the menu after selecting "start" in menu. It is normal,don't panic.
+3. Connect your phone to PC/Laptop and get your stock boot.img and then write this while device is on menu:  
+-`fastboot flash boot stock_boot.img`  
+After this,you should be able to reboot into Android and keep the lk2nd.
```

Comment:
I don't understand why this phone needs special installation... I guess you have to flash the recovery via heimdall/Odin as well?

Reply:
Yes,I have used both Heimdall and Odin for installation.